onyo/cli/tests/test_unset.py

```diff
@@ -279,7 +279,7 @@ def test_unset_message_flag(repo: OnyoRepo,
     ["num", "type"]])
 def test_unset_error_unset_name_fields(repo: OnyoRepo,
                                        name_field: list[str]) -> None:
-    """Test that `onyo unset KEY <asset>` throws the correct error without printing the usual
+    r"""Test that `onyo unset KEY <asset>` throws the correct error without printing the usual
```

Comment:
"throws" -> "raises" would be probably the better word. (I have a suspection that this was written by myself :P)

Reply:
Thanks for the catch. I have fixed it locally, but will not include it in this PR.